lib/ClangImporter/SwiftDeclSynthesizer.cpp

```diff
@@ -2214,6 +2214,9 @@ SwiftDeclSynthesizer::makeOperator(FuncDecl *operatorMethod,
   newParams.push_back(lhsParam);
 
   for (auto param : *paramList) {
+    if (param->getParameterName().empty()) {
+      param->setName(ctx.getIdentifier("other"));
+    }
```

Comment:
Since this is changing the name of the parameter of the backing C++ function, can we clone the parameter using `ParamDecl::clone`, and then change the name of the cloned parameter? I think this would be a good thing to do even if the name doesn't need to be changed, i.e. when `if (param->getParameterName().empty())` doesn't trigger.